php/utils.php

```diff
@@ -215,7 +215,7 @@ function handle_exception($exception) {
  * @return [boolean] $is_hermes True if Terminus is operating on Hermes
  */
 function is_hermes() {
-  $is_hermes = (TERMINUS_HOST == 'dashboard.getpantheon.com');
```

Comment:
Grep found the old domain in three places:
php/commands/site.php
php/terminus.php
✓php/utils.php